packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -22,11 +22,16 @@ export class MenuItem extends FASTElement {
     public checked: boolean;
 
     public handleMenuItemKeyDown = (e: KeyboardEvent): boolean => {
+        console.log("hello from menu item...");
```

Comment:
Extraneous console log.

Reply:
Also not the whole file not meant to be in this PR, dangit!